web-app/src/main/webapp/css/style.css

```diff
@@ -130,16 +126,30 @@ label {
 }
 
 .small-button {
+  background-size: 90px 90px;
   bottom: 45px;
   height: 90px;
   position: absolute;
   width: 90px;
 }
 
 #menu-button {
-  left: 0;
+  background-image: url(/images/options-menu.svg);
+  left: 50px;
 }
 
 #toggle-button {
-  right: 0;
+  background-image: url(/images/toggle-view.svg);
+  right: 50px;
+}
+
+/* Accessibility */
+.sr-only:not(:focus):not(:active) {
```

Comment:
Q: what does this do? I've never seen this before :O

Reply:
I found the code [online](https://www.sarasoueidan.com/blog/accessible-icon-buttons/) but I think it makes it visually hidden for users using a mouse but not if they're using a keyboard? This [stackoverflow post](https://stackoverflow.com/questions/34175774/what-is-the-use-of-sr-only-focusable-class-in-bootstrap) explains what I was trying to do!